.github/ISSUE_TEMPLATE/general-issue.md

```diff
@@ -0,0 +1,30 @@
+---
+name: General Issue
+about: Cross cutting korg issues, policy issues, etc
```

Comment:
nit: instead of korg, maybe just org or Kubernetes org? Just to try to avoid as much jargon as possible :)